docs/source/cli/graph-construction/single-machine-gconstruct.rst

```diff
@@ -169,3 +171,55 @@ Below shows an example that contains one node type and an edge type. For a real
+- **-\-val-pct**: a float value (\>0. and \<1.) with default value ``0.1``. You can set this value to control the percentage of nodes/edges for validation. 
+
+.. Note::
+    The sum of the **train-pct** and **val-pct** should be less than 1. And the percentage of test nodes/edges is the result of 1-(train_pct + val_pct).
+
+- **-\-add-reverse-edges**: if add this argument, will add reverse edges to the given graph.
+- **-\-retain-original-features**: boolean value to control if use the original features generated by dataset, e.g., embeddings of paper abstracts. If set to ``true``, will keep the original features; otherwise we will use the tokenized text for using BERT models to generate embeddings.
+- **-\-num-parts**: (**Required**) integer value that specifies partitions the DGL graph to be split. Remember this number because we will need to set it in the model training step.
+- **-\-output**: (**Required**) the folder path that the partitioned DGL graph will be saved.
+
+`partition_graph_lp.py <https://github.com/awslabs/graphstorm/blob/main/tools/partition_graph_lp.py>`_ arguments
+------------------------------------------------------------------------------------------------------------------------------------
+- **-\-dataset**: (**Required**) the graph name defined for the saved DGL graph file.
+- **-\-filepath**: (**Required**) the file path of the saved DGL graph file.
+- **-\-target-etypes**: (**Required**) the canonical edge type for making prediction.  GraphStorm supports **one** predict edge type only. The format is ``src_ntype,etype,dst_ntype``, e.g., `"author,write,paper"`.
+- **-\-train-pct**: a float value (\>0. and \<1.) with default value ``0.8``. If you want the partition script to split nodes/edges for training/validation/test sets, you can set this value to control the percentage of nodes/edges for training.
+- **-\-val-pct**: a float value (\>0. and \<1.) with default value ``0.1``. You can set this value to control the percentage of nodes/edges for validation. 
+
+.. Note:: 
+    The sum of the **train-pct** and **val-pct** should less than 1. And the percentage of test nodes/edges is the result of 1-(train_pct + val_pct).
+
+- **-\-add-reverse-edges**: if add this argument, will add reverse edges to the given graphs.
+- **-\-train-graph-only**: boolean value to control if partition the training graph or not, default is ``true``.
+- **-\-retain-original-features**: boolean value to control if use the original features generated by dataset, e.g., embeddings of paper abstracts. If set to ``true``, will keep the original features; otherwise we will use the tokenized text for using BERT models to generate embeddings.
+- **-\-retain-etypes**: the list of canonical edge type that will be retained before partitioning the graph. This might be helpful to remove noise edges in this application. Format example: ``—-retain-etypes query,clicks,asin query,adds,asin query,purchases,asin asin,rev-clicks,query``.
```

Comment:
```suggestion
- **-\-retain-etypes**: the list of canonical edge type that will be retained before partitioning the graph. This might be helpful to remove noisy edge types in a graph. Format example: ``—-retain-etypes query,clicks,asin query,adds,asin query,purchases,asin asin,rev-clicks,query``.
```

Reply:
Removed the `train-graph-only`, `retain-original-features`, `retain-etypes`.